gitopsaddon/test-cycle-eks-ocp.sh

```
@@ -538,9 +795,39 @@ for cycle in $(seq 1 "$MAX_CYCLES"); do
     # blocking app dispatch indefinitely.
     if [ "$cycle" -lt "$MAX_CYCLES" ]; then
         log "Resetting between cycles..."
+
+        # Delete stale CSRs to prevent "too many CSR" throttling
         hub get csr -o name 2>/dev/null | grep gitops-addon | xargs hub delete --ignore-not-found 2>/dev/null || true
```

Shellcheck (0.11.0)

[warning] Shell functions can't be passed to external commands. Use separate script or sh -c.

(SC2033)